bigbluebutton-html5/imports/ui/components/video-provider/video-provider-graphql/video-list/video-list-item/user-status/component.tsx

```diff
@@ -25,7 +25,7 @@ const UserStatus: React.FC<UserStatusProps> = (props) => {
   return (
     <div>
       {away && !raiseHand && <span>⏰</span>}
```

Comment:
```suggestion
      {away && <span>⏰</span>}
```

I think you could apply the same idea for `away` icon, right?